src/Tribe/Promoter/Observer.php

```diff
@@ -64,16 +68,32 @@ public function hook() {
 	 * @since 4.10.1.2
 	 *
 	 * @param $attendee_id
+	 *
+	 * @deprecated TBD
+	 *
 	 */
```

Comment:
:no_entry_sign: Additional blank lines found at end of doc comment

<sub>hash: 530bbed8d898a4d78bdf30e3bc392df1</sub>